Makefile

```diff
@@ -803,6 +803,7 @@ verification: $(VERIFICATION) $(VERIFICATION_ARTIFACTS) current | $(BUILD_DIR)
 	$(QUIET): "$@: Begin: EVE_REL=$(EVE_REL), HV=$(HV), LINUXKIT_PKG_TARGET=$(LINUXKIT_PKG_TARGET)"
 	cp images/*.yml $|
 	cp pkg/verification/runme.sh pkg/verification/build.yml $|
+	cp -r $|/verification/* $|/installer
 	cp -r $|/installer/* $|/verification
```

Comment:
The cp a b; cp b a
looks odd to me.
Are you saying that the verification and the installer directory should be identical?
Then what happens if someone does a combined "make verification installer"? Wil the installer contain the verification image?

As I said, confusing.

Reply:
I do this because I must maintain the `grub.cfg` of verification (when we copy all the contents of the installer directory, the `grub.cfg` is being replaced, which causes issues). I will try to change it to make it easier.

Reply:
But it makes no sense. What should go in the installer directory is what is used to build the installer directory. And what should go in the verification directory is what is used to build the verificatiion image. Hence they must be kept separate when there are different files or different versions of files.